packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -89,7 +92,6 @@ contract Bridge is EssentialContract, IBridge {
     /// @param _addressManager The address of the {AddressManager} contract.
     function init(address _addressManager) external initializer {
         __Essential_init(_addressManager);
-        _ctx.msgHash == bytes32(PLACEHOLDER);
```

Comment:
This code should of course have been `_ctx.msgHash = bytes32(PLACEHOLDER);`, but it doesn't really matter except a bit of extra gas cost for the first tx that uses it. Both `bytes32(PLACEHOLDER)` and `bytes32(0)` cases were handled correctly in the code.